<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>-1</y>
      <width>761</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="3">
      <widget class="QLineEdit" name="Army2LineEdit"/>
     </item>
     <item row="10" column="3">
      <widget class="QLineEdit" name="Army2WiTargetLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Enter army 1 name:&lt;b style=&quot;color:red;&quot;&gt;*&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QLineEdit" name="Army1WinTargetLineEdit">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Enter amount of modelings:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLineEdit" name="AmountLineEdit">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Enter army 2 name:&lt;b style=&quot;color:red;&quot;&gt;*&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QPushButton" name="AddItemButton">
       <property name="text">
        <string>Add Item</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QLineEdit" name="SaveArmiesStateLineEdit"/>
     </item>
     <item row="7" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Army 1 foes units health to win: </string>
       </property>
      </widget>
     </item>
     <item row="11" column="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Save armies state every entered round:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Army 2 foes units health to win: </string>
       </property>
      </widget>
     </item>
     <item row="10" column="4">
      <widget class="QPushButton" name="AddCircumstanceButton">
       <property name="text">
        <string>Add circumstance</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="Army1LineEdit"/>
     </item>
     <item row="0" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Armies</string>
       </property>
      </widget>
     </item>
     <item row="14" column="4">
      <widget class="QPushButton" name="AddArmyButton">
       <property name="text">
        <string>Add army</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QPushButton" name="AddUnitButton">
       <property name="text">
        <string>Add unit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" rowspan="14" colspan="2">
      <widget class="QListWidget" name="ArmiesListWidget"/>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="ChangeCircumstancesButton">
       <property name="text">
        <string>Change circumstances</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="StartButton">
       <property name="styleSheet">
        <string notr="true">background-color:#F0FFFF;
font-weight:800;
color:black;
</string>
       </property>
       <property name="text">
        <string>Start Modeling</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="ChangeReinfButton">
       <property name="text">
        <string>Change reinforcements</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
